src/repo.rs

```diff
@@ -831,7 +831,7 @@ impl Repository {
 
     /// Make the repository HEAD directly point to the commit.
     ///
-    /// If the provided committish cannot be found in the repository, the HEAD
+    /// If the provided commitish cannot be found in the repository, the HEAD
     /// is unaltered and an error is returned.
     /// If the provided commitish cannot be peeled into a commit, the HEAD is
```

Comment:
e.g. here we have both spellings directly one after another - so maybe it makes sense to harmonize it? Or does it not make any sense because it we might get both variants back anyway?